test/shared/src/main/scala/zio/test/TestAspect.scala

```diff
@@ -270,10 +270,11 @@ object TestAspect extends TimeoutVariants {
     }
 
   /**
-   * An aspect that retries a test until success, without limit, for use with
-   * flaky tests.
+   * An aspect that retries a test until success, with a limit specified by the
+   * test framework, for use with flaky tests.
    */
-  val flaky: TestAspectPoly = eventually
+  val flaky: TestAspectPoly =
```

Comment:
I'd add `def flaky(n: Int)` and then define `val flaky = flaky(100)`.

Reply:
Good idea!